soc/arm/realtek_bee/rtl87x2g/soc.c

```diff
@@ -188,6 +188,15 @@ static int rtk_register_update(void)
 	return 0;
 }
 
+/* Overrides the weak ARM implementation */
+void sys_arch_reboot(int type)
+{
+	ARG_UNUSED(type);
+
+	extern void WDG_SystemReset(int wdt_mode, int reset_reason);
+	WDG_SystemReset(0, type);
```

Comment:
Please check the "ARG_UNUSED" in line 194.

Reply:
I have checked the ARG_UNUSED on line 194 and made the necessary modifications.